### PR TITLE
Combobox dev app

### PR DESCRIPTION
**Pull request by nielsr98**
Added simple combobox example to dev app and added a function to set the text of the combobox if the value is a string.

### CODE REVIEW

File: src/cdk-experimental/combobox/combobox.ts [hunk 1]
```diff
@@ -143,6 +143,13 @@ export class CdkCombobox<T = unknown> implements OnDestroy, AfterContentInit {
 
     if (valueChanged) {
       this.panelValueChanged.emit(value);
+      this._setTextContent(value);
+    }
+  }
+
+  private _setTextContent(content: T) {
+    if (typeof content === 'string') {
```
Comment (jelbourn) [inline src/cdk-experimental/combobox/combobox.ts:151]: I would probably not check if it's exactly a string and just do ``textContent = `${content}`;``. We could then later add support for custom toString function.

File: src/dev-app/cdk-experimental-combobox/cdk-combobox-demo-module.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import {RouterModule} from '@angular/router';
+import {CdkComboboxModule} from '@angular/cdk-experimental/combobox';
+import {CdkComboboxDemo} from './cdk-combobox-demo';
+import {PanelContent} from './panel-content';
+
+@NgModule({
+    imports: [
+        CdkComboboxModule,
+        CommonModule,
+        RouterModule.forChild([{path: '', component: CdkComboboxDemo}]),
```
Comment (jelbourn) [inline src/dev-app/cdk-experimental-combobox/cdk-combobox-demo-module.ts:20]:
```suggestion
    imports: [
      CdkComboboxModule,
      CommonModule,
      RouterModule.forChild([{path: '', component: CdkComboboxDemo}]),
```

File: src/dev-app/cdk-experimental-combobox/panel-content.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+@Directive({
+    selector: '[panelContent]',
+    exportAs: 'panelContent',
+    host: {
+        'role': 'role',
+        '[id]': 'dialogId'
+    }
+})
+export class PanelContent<V> implements OnInit {
```
Comment (jelbourn) [inline src/dev-app/cdk-experimental-combobox/panel-content.ts:24]: Is this just for testing, or do you intend for this to be the ultimate API?

Comment (nielsr98) [reply src/dev-app/cdk-experimental-combobox/panel-content.ts:24]: So this directive is just for testing/demo purposes. It is the directive that is applied to the custom panel content the user wishes to use, like their own dialog. It handles registering the content with the panel. I did realize it was doing more than I needed it to do in this example so I've toned it down.

**Pull request merged**